distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso

```diff
@@ -129,6 +129,10 @@ type Comparable
 
     new value:Any comparator:Any -> Comparable = Comparable.By value comparator
 
+    ## PRIVATE
+    less_than_builtin : Any -> Any -> Boolean | Nothing
+    less_than_builtin = @Builtin_Method "Comparable.less_than_builtin"
```

Comment:
Please remove this method. `@Builtin_Method`s are supposed to be only in `private` modules. Moreover there already is such a `less_than_builtin` in `Ordering_Helpers.enso`.

Reply:
Ok, this method is indeed wrong, as I don't see a `Comparable.less_than_builtin` builtin in Java.

It was probably an artifact from some merges that happened along the way. I'll remove this one.